contracts/State.sol

```diff
@@ -47,85 +39,63 @@ contract State {
    * @param timestamp Timestamp when the state has been commited
    * @param state IDState commited
    */
-  event StateUpdated(bytes31 id, uint64 blockN, uint64 timestamp, bytes32 state);
+  event StateUpdated(uint256 id, uint64 blockN, uint64 timestamp, uint256 state);
```

Comment:
Identity used to be [31]byte.  Is the uint256 the big int representation of the Identity?